genesis.block

```diff
@@ -45,7 +47,15 @@
                     "role_name": "voter",
                     "permissions": [
                         "can_transfer",
-                        "can_get_all_acc_ast"
+                        "can_get_all_acc_ast",
+                        "can_get_my_acc_ast_txs"
```

Comment:
the previous perm should be enough

Reply:
I want voter to see only her own transactions (to check it was correct). I think can_get_all_acc_ast should be removed